### ISSUE TITLE
Network Bridge Operation not Supported

### ISSUE THREAD

**Issue by invertium** [cockpit-project/cockpit#14486]
Cockpit version: Version 225.
OS:
Page: https://stabl-trocken:9090/machines

I'm trying to add a new network interface to bridge to my network, however when I select the following options:
![image](https://user-images.githubusercontent.com/15930333/90506911-4ab3f980-e155-11ea-9e56-7b90bd6719eb.png)
I get Unable to add bridge enp3s0 port vnet0: Operation not supported

Steps to reproduce

1. Create new VM (Ubuntu Server 19.04) on Debian
2. Create New Network Interface with the options above
3. Try to start machine

**Comment by marusak** [comment]
Indeed, I can reproduce. However no idea what it means for us. Should we not show it in the UI if we cannot support it? Or should just user pick something different then? @KKoukiou might have much better insight.

![notsupported](https://user-images.githubusercontent.com/12330670/91954530-3699fb80-ed02-11ea-9a1f-e08d02a4679d.png)

**Comment by aaroncnb** [comment]
Can reproduce this issue on my Fedora Server 33 machin, when trying to add a network bridge to my Ubuntu 18.04 Server virtual machine.

Adding the interface while the VM is powered-off intially appears successful, but the error message (same as reported by @marusak and @invertium ) appears when I try to Run the VM:

![image](https://user-images.githubusercontent.com/12600975/100071785-ca511300-2e7e-11eb-914d-f0789680c09d.png)

**Comment by invertium** [comment]
A fix/implementation would be awesome, easy firing up of VMs is a real AWS like feature 👍

**Comment by skobyda** [comment]
Yes, the enp3s* interface should not be even in that list. Here user should be ably to only add a bridge as an interface to a VM. That bridge, in turn, would have a physical interface (e.g. enp3s*) added to them. You can create such bridge in Networking page of cockpit and then use this bridge for the Virtual machine networking.
So as a fix, I see that physical interfaces should not even be present in "Source" list for "Bridge to LAN" interface type. And we can use the functionality from Networking page to be used directly at machines page to create a bridge which then user can use for a VM.
